search/includes/classes/class-search.php

```diff
@@ -63,6 +63,74 @@ class Search {
 		'advanced_seo_description',
 	);
 
+	public const ES_QUERY_RESERVED_NAMES = array(
```

Comment:
Just as an idea — it does not necessarily go into this PR. We have this (or a similar to this) list in WordPress:

https://github.com/WordPress/WordPress/blob/5.8.2/wp-includes/class-wp.php#L17
https://github.com/WordPress/WordPress/blob/5.8.2/wp-includes/class-wp.php#L27

Does it make sense to generate our stop list from those WP lists? The advantage of this (as I see it) is that the code will be future-proof — if WP adds another reserved query variable, we won't have to update our code.

Reply:
Oh that is a great idea!,,, I'll give it a try